### ISSUE TITLE
Gradle plugin

### ISSUE THREAD

**Issue by jacek99** [dodie/scott#44]
It would be great if there was a Gradle plugin, Maven has been falling out of favor with many shops...thanks!

**Comment by dodie** [comment]
Great idea!

**Comment by dodie** [comment]
Most projects I work on use Maven, but Gradle gained a lot of ground lately. A plugin would definitely help to be more useful for more projects.

**Comment by dodie** [comment]
To get started, read the current [How to use guide](https://github.com/dodie/scott#how-to-use) and check the [Maven Plugin](https://github.com/dodie/scott/tree/master/scott-maven-plugin) (issues [#4](https://github.com/dodie/scott/issues/4) and [#39](https://github.com/dodie/scott/issues/39)).

**Comment by luksrn** [comment]
Can I help with this task @dodie ? Yesterday I started the research process to understand how the plugin can be developed. The references you have passed are a good starting point! : D

I believe that I will send news soon to align the development and clarify doubts: D

**Comment by dodie** [comment]
Hi @luksrn ! Sure, sounds awesome! :)

**Comment by luksrn** [comment]
Hey @dodie , I've researched how to create custom plugins and I've already made some interesting headway.

Basically, in addition to reading some documentation guides, I used the Jacoco Plugin as the basis for the implementation.

I still have some tests with JUnit 5 and Cocumber, but I have already been able to validate with Junit4. The documentation is also missing.

In a second moment, I go into more detail, it's too late here! 3 o'clock in the morning. HAHA HA
